fetch/linux/oratab.sh

```diff
@@ -24,7 +24,7 @@ fi
 
 if [ ! -r "$ORATAB" ]; then
   >&2 echo "The file $ORATAB doesn't exist or isn't readable"
-  exit 1
+  exit 0
```

Comment:
Why? Exit code should be 0 if there are no error, but this is an error.

Reply:
Do you not have opened a issue that say ercole-agent should not fail if oratab is not readable (#133) ?

Reply:
Not exactly: it should execute successfully without any db installed, not if the oratab is not readable.
In fact, it was possible now disabling OracleDatabase feature in config.json.
This functionality was added later and that issue could already be closed. Amen.